### PR TITLE
PUP-9088 Added runtime support for Ubuntu 18.10 amd64

### PR DESCRIPTION
**Pull request by sebastian-miclea**
Added a new platform for Ubuntu 18.10 amd64, in order to be able to build PA.

The template will be available after the the packer pull request is merged.
https://github.com/puppetlabs/puppetlabs-packer/pull/364

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ script:
 - |
   stat=0
   for projfile in $(ls configs/projects/[a-z]*.rb); do
-    for plat in el-7-x86_64 ubuntu-18.04-amd64 osx-10.13-x86_64 windows-2012r2-x64; do
+    for plat in el-7-x86_64 ubuntu-18.04-amd64 ubuntu-18.10-amd64 osx-10.13-x86_64 windows-2012r2-x64; do
```
Comment (ekinanp) [inline .travis.yml]: Do we need to add it here? We already have a representative Ubuntu platform. Won't block the PR on this, just curious is all.

File: configs/platforms/ubuntu-18.10-amd64.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "make",
+    "pkg-config",
+    "pl-cmake",
+    "pl-gcc",
+    "zlib1g-dev"
+  ]
+
+  plat.provision_with "export DEBIAN_FRONTEND=noninteractive; apt-get install -qy --no-install-recommends #{packages.join(' ')}"
+
+  plat.vmpooler_template "ubuntu-1804-x86_64"
```
Comment (ekinanp) [inline configs/platforms/ubuntu-18.10-amd64.rb]: This should be 1810

File: .travis.yml (hunk 1 above)
Comment (ScottGarman) [reply .travis.yml]: Yeah, no need to add the platform to travis.

**Comment by ScottGarman** [comment]
It looks like this PR needs to be rebased against the latest master.

**Pull request closed without merging**